common/common.go

```diff
@@ -28,6 +28,24 @@ const (
 	HashLength = 32
 )
 
+// StringArrayToBytes turns an array of strings into an array of byte arrays
+func StringArrayToBytes(in []string) [][]byte {
```

Comment:
What are these needed for? It seems their being used in place of SCALE?

Reply:
it was used for turning the bootstrapNodes struct field into arrays of byte arrays and back to strings, since strings weren't working. if we fixed them then this isn't needed